### PR TITLE
Update license and attribution to Plaid Technologies Inc

### PR DESCRIPTION
**Pull request by pbernasconi**
@j4ustin –the Plaid team would like to thank you for your contributions to the plaid-ruby library.

We are simply correcting the copyright attribution to reflect that ownership.  Thanks again, so much, for your contributions! We have attributed your name in the README, as noted below.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -123,3 +123,8 @@ Learn about the full functionality of the library on our [Wiki](https://github.c
 We highly encourage helping out with the gem. Either adding more tests, building new helper classes, fixing bugs, or anything to increase overall quality.
 
 Learn more about best practices, submitting a pull request, and rules for the build on our [Wiki](https://github.com/plaid/plaid-ruby/wiki/Contribute!)
+
+
+## Contributors
+
+- @j4ustin (Justin Crites)
```
Comment (davidchambers) [inline README.md]: Let's have the username link to Justin's GitHub profile. :)

File: README.md [hunk 2]
```diff
@@ -123,3 +123,8 @@ Learn about the full functionality of the library on our [Wiki](https://github.c
 We highly encourage helping out with the gem. Either adding more tests, building new helper classes, fixing bugs, or anything to increase overall quality.
 
 Learn more about best practices, submitting a pull request, and rules for the build on our [Wiki](https://github.com/plaid/plaid-ruby/wiki/Contribute!)
+
+
```
Comment (davidchambers) [inline README.md]: :space_invader:

**Comment by j4ustin** [comment]
@pbernasconi @davidchambers thanks guys! Glad to help. 

If you need any help on the `golang` version let me know, I've been doing (more than) full time work in it for over a year.

**Comment by davidchambers** [comment]
Thank you for your offer, @j4ustin! We'll get in touch if we decide to take you up on it. :)

File: LICENSE [hunk 3]
```diff
@@ -1,6 +1,6 @@
-Copyright (c) 2015 Justin Crites
+The MIT License
 
-MIT License
+Copyright (c) 2013-present Plaid Technologies, Inc. <support@plaid.com>
```
Comment (davidchambers) [inline LICENSE]: `s/2013-present/2016/`

Comment (pbernasconi) [inline LICENSE]: ⚡

File: LICENSE [hunk 4]
```diff
@@ -1,6 +1,6 @@
-Copyright (c) 2015 Justin Crites
+The MIT License
 
-MIT License
+Copyright (c) 2013-2016 Plaid Technologies, Inc. <support@plaid.com>
```
Comment (davidchambers) [inline LICENSE]: 2013? The [initial commit][1] was in 2015.


[1]: https://github.com/plaid/plaid-ruby/commit/2ee9df00626e5e9213fcfe34c3161ea5d2e3a013

File: README.md [hunk 5]
```diff
@@ -123,3 +123,7 @@ Learn about the full functionality of the library on our [Wiki](https://github.c
 We highly encourage helping out with the gem. Either adding more tests, building new helper classes, fixing bugs, or anything to increase overall quality.
 
 Learn more about best practices, submitting a pull request, and rules for the build on our [Wiki](https://github.com/plaid/plaid-ruby/wiki/Contribute!)
+
+## Contributors
+
+- @j4ustin (Justin Crites)
```
Comment (davidchambers) [inline README.md]: Let's link to Justin's GitHub profile.

**Comment by be9** [comment]
As of 2.0.0.alpha, the LICENSE has this line:

> Copyright (c) 2016 Plaid Technologies, Inc.

Is it OK?

**Comment by davidchambers** [comment]
That's perfect, @be9!

**Comment by be9** [comment]
Great, closing then.

**Pull request closed without merging**